pandas/tests/test_strings.py

```diff
@@ -965,6 +965,31 @@ def test_pad_fillchar(self):
         with tm.assertRaisesRegexp(TypeError, "fillchar must be a character, not int"):
             result = values.str.pad(5, fillchar=5)
 
+    def test_translate(self):
```

Comment:
for the first one, you mean testing `s.str.translate()` when some of the strings are empty?